src/extensions/action_items.py

```diff
@@ -0,0 +1,111 @@
+import arc
+import hikari
+import requests
+import re
+
+# not used, functionality for times/deadlines to be added later
+from datetime import datetime, timezone
+
+from src.utils import role_mention
+from src.hooks import restrict_to_channels, restrict_to_roles
+from src.config import CHANNEL_IDS, ROLE_IDS, UID_MAPS, LDAP_USERNAME, LDAP_PASSWORD
+
+
+action_items = arc.GatewayPlugin(name="Action Items")
+
+session = requests.Session()
```

Comment:
Need to use `aiohttp` instead of `requests` as requests is blocking (not async).

I will push a commit to change this as I want to do it a specific way.